run_dir/design/base.html

```diff
@@ -57,10 +57,8 @@
                       <li class="dropdown-header">General stats</li>
                       <li><a href="/applications">Application Counts</a></li>
                       <li><a href="/production/cronjobs">Cronjobs List</a></li>
-                      <li role="separator" class="divider"></li>
-                      <li class="dropdown-header">Flowcells</li>
                       <li><a href="/flowcells_plot">Flowcells Trends</a></li>
-                      <li><a href="/flowcell_count_plot">Flowcells Numbers</a></li>
+                      <li><a href="/data_delivered_plot">Data Delivered Trends</a></li>
```

Comment:
You will have to make the corresponding changes to the base_b5.html file as well.